aptos-move/framework/aptos-framework/sources/transaction_validation.spec.move

```diff
@@ -82,42 +82,42 @@ spec aptos_framework::transaction_validation {
         let transaction_sender = signer::address_of(sender);
         let gas_payer_addr = signer::address_of(gas_payer);
 
-        aborts_if (
-            !features::spec_is_enabled(features::SPONSORED_AUTOMATIC_ACCOUNT_CREATION)
-                || account::spec_exists_at(transaction_sender)
-                || transaction_sender == gas_payer_addr
-                || txn_sequence_number > 0
-        ) && (
-            !(txn_sequence_number >= global<Account>(transaction_sender).sequence_number)
-                || !(option::spec_is_none(txn_authentication_key) || option::spec_borrow(
-                txn_authentication_key
-            ) == global<Account>(transaction_sender).authentication_key)
-                || !account::spec_exists_at(transaction_sender)
-                || !(txn_sequence_number == global<Account>(transaction_sender).sequence_number)
-        );
-
-        aborts_if features::spec_is_enabled(features::SPONSORED_AUTOMATIC_ACCOUNT_CREATION)
-            && transaction_sender != gas_payer_addr
-            && txn_sequence_number == 0
-            && !account::spec_exists_at(transaction_sender)
-            && (option::spec_is_none(txn_authentication_key) || option::spec_borrow(
-            txn_authentication_key
-        ) != bcs::to_bytes(transaction_sender));
-
-        aborts_if !(txn_sequence_number < (1u64 << 63));
-
-        let max_transaction_fee = txn_gas_price * txn_max_gas_units;
-        aborts_if max_transaction_fee > MAX_U64;
-        aborts_if !exists<CoinStore<AptosCoin>>(gas_payer_addr);
-        // property 1: The sender of a transaction should have sufficient coin balance to pay the transaction fee.
-        /// [high-level-req-1]
-        aborts_if !(global<CoinStore<AptosCoin>>(gas_payer_addr).coin.value >= max_transaction_fee);
+        // aborts_if (
```

Comment:
we either fix it or remove it, having it commented out looks awkward

Reply:
I'm not sure that's how we maintain the spec, are we still actively maintaining it? @rahxephon89

Reply:
Yes, we plan to fix these broken specs and it is OK to remove them temporarily.